CTF/the_simpsons.md

```diff
@@ -12,13 +13,55 @@ Ya know, I was thinking... wouldn't the Simpsons use octal as a base system? The
 3. at the button there are lines of python code 
-4. decode the encoded and key using octal mentioned in the description 
-5. get encoded as 'jrjerwhzkrexar', get key as 'How much did Maggie originally cost? (Divided by 8, to the nearest integer, and then plus four)'
+   the python code states that: 
+   
+   encoded = 152 162 152 145 162 167 150 172 153 162 145 170 141 162
+   key = chr(SolutionToDis(110 157 167 040 155 165 143 150 040 144 151 144 040 115 141 147 147 151 145 040 157 162 151 147 151    156 141 154 154 171 040 143 157 163 164 077 040 050 104 151 166 151 144 145 144 040 142 171 040 070 054 040 164 157 040 164    150 145 040 156 145 141 162 145 163 164 040 151 156 164 145 147 145 162 054 040 141 156 144 040 164 150 145 156 040 160 154    165 163 040 146 157 165 162 051)). 
+   key = key + key + chr(ord(key)-4). 
+   print(DecodeDat(key=key,text=encoded))
+   
+4. decode the encoded and key from the code above using octal
+   this is done by using cyberchef, choose "from octal"
+   why use octal? -- becuase it is mentioned by the description : "wouldn't the Simpsons use octal as a base system" 
+   then put that series of numers (152 162 152 145 162 167 150 172 153 162 145 170 141 162) as input 
+   
+5. After decoding the encoded text above, you would get text like `jrjerwhzkrexar`
+   it suggests that after converting this series of number from octal, it returns you the original text, which is        `jrjerwhzkrexar`
+   website of cyberchef: https://gchq.github.io/CyberChef/#recipe=From_Octal('Space')&input=MTUyIDE2MiAxNTIgMTQ1IDE2MiAxNjcgMTUwIDE3MiAxNTMgMTYyIDE0NSAxNzAgMTQxIDE2Mg
+   
+6. After which you can get key using the instructions as stated in the hex-dumped value of the picture. The instructions are as follows:
+- 'How much did Maggie originally cost? 
+- Divided by 8, to the nearest integer, and then plus four)'
+
 6. search the ans of the qn in key online and do calculation get 110
+   for the answer of the first question, it can be found on this website:
+   https://mashable.com/2014/08/21/simpsons-facts/#:~:text=1.,raising%20a%20child%20in%201989.
+   it gives $847.67
+   then 847.67 / 8, to the nearest integer, it will be 110
+   
 7. follow the key - use chr(the result of the key) get key = n
+   as the code states: chr(the solution) 
+   chr means, convert the number to letter according to the ACSII table:
+   the table can be found here : http://www.asciitable.com/
+   the letter returned is n 
+   
 8. then the line below gives a new value of key = key + key + chr(ord(key)-4), get new key as nnj
-9. finally, use the new key to decipher the encoded text "jrjerwhzkrexar" 
+   key is assigned to key + key + chr(ord(key)-4)
+   key + key = n + n = nn
+   then chr(ord(key)-4):
+   ord(key) means to convert the key vvalue back to the number according to the ascii table
+   so it gives value as 110, then 110 - 4 = 106
+   so chr(ord(key)-4) == chr(106) == j
+   so key = nnj 
```

Comment:
@HS-O this is a markdown file hence just giving the code a newline wont actually give it a newline. please check up on how to do markdown thanks!